synapse/lib/storm.py

```diff
@@ -14,6 +15,25 @@
 
 logger = logging.getLogger(__name__)
 
+class LimitHelp:
+
+    def __init__(self, limit):
+        self.limit = limit
```

Comment:
it needs to be ok for limit to be None ( our helper makes the user not need to worry about limit=None case )